test/drbgtest.c

```diff
@@ -983,84 +983,84 @@ static int test_multi_thread(void)
 
     if (crngt_skip())
         return TEST_skip("CRNGT cannot be disabled");
 
     /* Initialise a three long DRBG chain */
-    if (!TEST_ptr(m = RAND_DRBG_new(0, 0, NULL))
-        || !TEST_true(disable_crngt(m))
-        || !TEST_true(RAND_DRBG_instantiate(m, NULL, 0))
-        || !TEST_ptr(i = RAND_DRBG_new(0, 0, m))
-        || !TEST_true(RAND_DRBG_instantiate(i, NULL, 0))
-        || !TEST_ptr(s = RAND_DRBG_new(0, 0, i))
-        || !TEST_true(RAND_DRBG_instantiate(s, NULL, 0)))
+    if (!TEST_ptr(x = RAND_DRBG_new(0, 0, NULL))
+        || !TEST_true(disable_crngt(x))
+        || !TEST_true(RAND_DRBG_instantiate(x, NULL, 0))
+        || !TEST_ptr(y = RAND_DRBG_new(0, 0, x))
+        || !TEST_true(RAND_DRBG_instantiate(y, NULL, 0))
+        || !TEST_ptr(z = RAND_DRBG_new(0, 0, y))
+        || !TEST_true(RAND_DRBG_instantiate(z, NULL, 0)))
         goto err;
 
-    /* During a normal reseed, only the slave DRBG should be reseed */
-    inc_reseed_counter(i);
-    mreseed = reseed_counter(m);
-    ireseed = reseed_counter(i);
-    sreseed = reseed_counter(s);
-    if (!TEST_true(RAND_DRBG_reseed(s, NULL, 0, 0))
-        || !TEST_int_eq(reseed_counter(m), mreseed)
-        || !TEST_int_eq(reseed_counter(i), ireseed)
-        || !TEST_int_gt(reseed_counter(s), sreseed))
+    /* During a normal reseed, only the leaf DRBG should be reseeded */
+    inc_reseed_counter(y);
+    xreseed = reseed_counter(x);
+    yreseed = reseed_counter(y);
+    zreseed = reseed_counter(z);
+    if (!TEST_true(RAND_DRBG_reseed(z, NULL, 0, 0))
+        || !TEST_int_eq(reseed_counter(x), xreseed)
+        || !TEST_int_eq(reseed_counter(y), yreseed)
+        || !TEST_int_gt(reseed_counter(z), zreseed))
         goto err;
 
     /*
      * When prediction resistance is requested, the request should be
-     * propagated to the master, so that the entire DRBG chain reseeds.
+     * propagated to the primary, so that the entire DRBG chain reseeds.
      */
-    sreseed = reseed_counter(s);
-    if (!TEST_true(RAND_DRBG_reseed(s, NULL, 0, 1))
-        || !TEST_int_gt(reseed_counter(m), mreseed)
-        || !TEST_int_gt(reseed_counter(i), ireseed)
-        || !TEST_int_gt(reseed_counter(s), sreseed))
+    zreseed = reseed_counter(z);
+    if (!TEST_true(RAND_DRBG_reseed(z, NULL, 0, 1))
+        || !TEST_int_gt(reseed_counter(x), xreseed)
+        || !TEST_int_gt(reseed_counter(y), yreseed)
+        || !TEST_int_gt(reseed_counter(z), zreseed))
         goto err;
 
-    /* During a normal generate, only the slave DRBG should be reseed */
-    inc_reseed_counter(i);
-    mreseed = reseed_counter(m);
-    ireseed = reseed_counter(i);
-    sreseed = reseed_counter(s);
-    if (!TEST_true(RAND_DRBG_generate(s, buf1, sizeof(buf1), 0, NULL, 0))
-        || !TEST_int_eq(reseed_counter(m), mreseed)
-        || !TEST_int_eq(reseed_counter(i), ireseed)
-        || !TEST_int_gt(reseed_counter(s), sreseed))
+    /* During a normal generate, only the leaf DRBG should be reseed */
```

Comment:
```suggestion
    /* During a normal generate, only the secondary DRBGs should be reseed */
```

Reply:
Same comment as before but the clarification is simplified to _last_.